addOns/pscanrules/CHANGELOG.md

```diff
@@ -7,34 +7,35 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Changed
 - The Big Redirect scan rule will now also alert on responses that have multiple HREFs (idea from xnl-h4ck3r).
 - The references for the following scan rules are now all HTTPS (Issue 8262) and in some cases updated:
-    - Loosely Scoped Cookie
-    - Charset Mismatch
-    - Strict-Transport-Security Header
-    - Content Security Policy (CSP) Header Not Set
-    - CSP
-    - Session ID in URL Rewrite
-    - HTTP Server Response Header
-    - Cookie Poisoning
-    - User Controllable HTML Element Attribute (Potential XSS)
-    - X-Content-Type-Options Header Missing
-    - Content-Type Header Missing
-    - Server Leaks Information via "X-Powered-By" HTTP Response Header Field(s)
+  - Loosely Scoped Cookie
+  - Charset Mismatch
+  - Strict-Transport-Security Header
+  - Content Security Policy (CSP) Header Not Set
+  - CSP
+  - Session ID in URL Rewrite
+  - HTTP Server Response Header
+  - Cookie Poisoning
+  - User Controllable HTML Element Attribute (Potential XSS)
+  - X-Content-Type-Options Header Missing
+  - Content-Type Header Missing
+  - Server Leaks Information via "X-Powered-By" HTTP Response Header Field(s)
```

Comment:
These whitespace changes should be reverted.

Reply:
ok having a look.

Reply:
done